### PR TITLE
Update addon to deal with phabricator changes

### PR DESCRIPTION
**Pull request by staktrace**
The structure of the rendered HTML has changed slightly:
- There's no `<th>` with the line number anymore, it's a `<td>` with a
  dataset now.
- The actual code is no longer broken up into elements, so we need to
  tokenize the text explicitly and then look up the tokens in the
  searchfox data

### CODE REVIEW

File: phabricator.js [hunk 1]
```diff
@@ -153,11 +153,31 @@ function addLinksAndHighlight(elem, searchfoxElem, searchfoxAnalysisData) {
...
 }
 
 let parsedLines = new Set();
 
+// Yields objects with start and end offsets, which define ASCII words
+// in the given text, from last to first.
+function* tokenOffsets(text) {
+  let tokenEnd = null;
+  for (var i = text.length - 1; i >= 0; i--) {
+    let charInToken = (text[i] >= 'A' && text[i] <= 'Z') ||
```
Comment (marco-c) [inline phabricator.js]: Couldn't a token contain a number too?

File: phabricator.js [hunk 2]
```diff
@@ -153,11 +153,31 @@ function addLinksAndHighlight(elem, searchfoxElem, searchfoxAnalysisData) {
...
+  return block.querySelectorAll('table.differential-diff tbody tr td.n');
 }
 
 let parsedLines = new Set();
 
+// Yields objects with start and end offsets, which define ASCII words
+// in the given text, from last to first.
+function* tokenOffsets(text) {
+  let tokenEnd = null;
+  for (var i = text.length - 1; i >= 0; i--) {
```
Comment (marco-c) [inline phabricator.js]: Nit: let i

**Comment by staktrace** [comment]
Patch updated to address comments.

**Review by marco-c — APPROVED** [review]
Thanks!

**Pull request merged**